activitypub/inbox/delete.go

```diff
@@ -0,0 +1,25 @@
+package inbox
+
+import (
+	"context"
+
+	"github.com/go-fed/activity/streams/vocab"
+	"github.com/owncast/owncast/activitypub/persistence"
+	"github.com/owncast/owncast/activitypub/resolvers"
+	log "github.com/sirupsen/logrus"
+)
+
+func handleDeleteRequest(c context.Context, activity vocab.ActivityStreamsDelete) error {
+	// Only need to handle deletes for followers
+	if !activity.GetActivityStreamsObject().At(0).IsActivityStreamsPerson() {
```

Comment:
I've probably used `At()` myself where I probably shouldn't, but if for some reason a `Delete` comes in what doesn't have an associated object attached this would crash.